src/d3d9/d3d9_common_texture.cpp

```diff
@@ -230,6 +289,14 @@ namespace dxvk {
     imageInfo.tiling          = VK_IMAGE_TILING_OPTIMAL;
     imageInfo.layout          = VK_IMAGE_LAYOUT_GENERAL;
     imageInfo.shared          = m_desc.IsBackBuffer;
+    if (pSharedHandle) {
+      imageInfo.sharing.type = VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_KMT_BIT;
+      if (*pSharedHandle) {
+        imageInfo.sharing.mode = *pSharedHandle == INVALID_HANDLE_VALUE ? DxvkSharedHandleMode::Export : DxvkSharedHandleMode::Import;
+        imageInfo.sharing.handle = *pSharedHandle;
+      }
+      // TODO: validate metadata?
```

Comment:
`imageInfo.shared = true;`?

Also need to keep shared images in `GENERAL` layout to be consistent with D3D11.